e2e/src/usecases/SecureSend.js

```diff
@@ -1,92 +1,175 @@
 import {
-  SAMPLE_BACKUP_KEY_VERIFIED,
+  SAMPLE_BACKUP_KEY_SINGLE_ADDRESS_VERIFIED,
+  SAMPLE_PRIVATE_KEY,
   VERIFIED_PHONE_NUMBER,
+  SAMPLE_WALLET_ADDRESS_SINGLE_ADDRESS_VERIFIED,
   SAMPLE_WALLET_ADDRESS_VERIFIED_2,
 } from '../utils/consts'
 import { launchApp } from '../utils/retries'
 import {
   addComment,
   enterPinUiIfNecessary,
+  fundWallet,
   inputNumberKeypad,
   scrollIntoView,
   quickOnboarding,
+  waitForElementId,
+  waitForElementByIdAndTap,
 } from '../utils/utils'
 const faker = require('@faker-js/faker')
```

Comment:
Faker can be removed here if it's no longer being used. Personally, I think it's worth having to verify the transaction in the home screen; however, that's not currently tested so it's fine to remove. We can add it back later if we want to assert on the comment used in the send.